test/fixtures.json

```diff
@@ -143,6 +143,32 @@
         "sha224": "3c198cbdb9464b7857966bd05b7bc92bc1cc4e6e63155d4e490557fd859894978ab846d52a1083ac610c36c2c5ea8ce4a024dd691064d5453bd17b15ea1ac1944bbfd62e61b997e7b22660f588e297186572480015f33bc2bfd2b423827bcdcdb4845914",
         "sha384": "c0e14f06e49e32d73f9f52ddf1d0c5c7191609233631dadd76a567db42b78676b38fc800cc53ddb642f5c74442e62be44d727702213e3bb9223c53b767fbfb5db9d270d54c45d9cb6003d2967280b22671e2dbc6375f6ebf219c36f0d127be35e19d65a8"
       }
+    },
+    {
+      "keyuint8": [112, 97, 115, 115, 119, 111, 114, 100],
+      "salt": "salt",
+      "iterations": 1,
+      "dkLen": 32,
+      "results": {
+        "sha1": "0c60c80f961f0e71f3a9b524af6012062fe037a6e0f0eb94fe8fc46bdc637164",
+        "sha256": "120fb6cffcf8b32c43e7225256c4f837a86548c92ccc35480805987cb70be17b",
+        "sha512": "867f70cf1ade02cff3752599a3a53dc4af34c7a669815ae5d513554e1c8cf252",
+        "sha224": "3c198cbdb9464b7857966bd05b7bc92bc1cc4e6e63155d4e490557fd85989497",
+        "sha384": "c0e14f06e49e32d73f9f52ddf1d0c5c7191609233631dadd76a567db42b78676"
+      }
+    },
+    {
+      "key": "password",
+      "saltuint8": [115, 97, 108, 116],
```

Comment:
I think a capitalization after `key` and `salt` is needed for readability. I suggest `keyUint8Array` and `saltUint8Array` below but any kind of upper case letter would probably do the trick